### PR TITLE
Purge attendees transient on checkin/uncheckin

### PR DESCRIPTION
**Pull request by juanfra**
https://central.tri.be/issues/73272

https://central.tri.be/issues/104355

Purge attendees transient when checkin or uncheckin is performed. By having this in the main events, will be compatible with the different e-commerce tickets.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Expected 1 space after comma in function call; 3 found | [Tickets.php:815](https://github.com/moderntribe/event-tickets/blob/90e5005bd167b4d4555f509d38ec250a0cf0ec95/src/Tribe/Tickets.php#L815)
:x: | Must use &quot;self::&quot; for local static member reference | [Tickets.php:830](https://github.com/moderntribe/event-tickets/blob/90e5005bd167b4d4555f509d38ec250a0cf0ec95/src/Tribe/Tickets.php#L830)

> via jenkins/codesniffer-phpcs

File: readme.txt [hunk 1]
```diff
@@ -123,6 +123,8 @@ Currently, the following add-ons are available for Event Tickets:
 
 = [4.7.4] TBD =
 
+* Fix - Properly update attendee's transient when checkin/unchekin an attendee, in order to see changes immediately. Thanks to @newcollegeofflorida and @gschnoor for flagging this! [73272]
```
Comment (Camwyn) [inline readme.txt]: No apostrophe in `attendees`

File: src/Tribe/Tickets.php [hunk 2]
```diff
@@ -811,6 +811,26 @@ public function __construct() {
 
 			// Ensure ticket prices and event costs are linked
 			add_filter( 'tribe_events_event_costs', array( $this, 'get_ticket_prices' ), 10, 2 );
+
+			add_action( 'event_tickets_checkin', array( $this, 'purge_attendees_transient' ) );
+			add_action( 'event_tickets_uncheckin', array( $this, 'purge_attendees_transient' ) );
+		}
+
+		/**
+		 * Remove the Post Transients when a Ticket change its state
```
Comment (Camwyn) [inline src/Tribe/Tickets.php]: I'd reword to "Remove the Post Transients when the Ticket state changes"
